InvenTree/common/models.py

```diff
@@ -875,6 +875,12 @@ def save(self, *args, **kwargs):
             ]
         },
 
+        'INVENTREE_DOWNLOAD_FROM_URL_USER_AGENT': {
+            'name': _('User-agent used to download from URL'),
+            'description': _('Allow to override the user-agent used to download mages and files from external URL (leave blank for the default)'),
```

Comment:
Missing letter for image

Reply:
@matmair  It seems typing is hard this morning... but I've added the missing letter